Dockerfile

```diff
@@ -31,6 +31,9 @@ RUN mkdir -p $LEIN_INSTALL \
   && rm leiningen-$LEIN_VERSION-standalone.zip.asc \
   && mv leiningen-$LEIN_VERSION-standalone.zip /usr/share/java/leiningen-$LEIN_VERSION-standalone.jar
 
+# Some REPLs (e.g., Figwheel) necessitate a readline wrapper.
+RUN apt-get update && apt-get install rlfe && rm -rf /var/lib/apt/lists/*
```

Comment:
@dmohs perhaps make use of the previous `RUN` like the other lines so we don't add another layer?